api/DOMRect.json

```diff
@@ -20,10 +20,10 @@
             }
           ],
           "firefox": {
-            "version_added": true
+            "version_added": "27"
```

Comment:
This seems odd, since no subfeature in https://developer.mozilla.org/en-US/docs/Web/API/DOMRect#Browser_compatibility was introduced in Firefox 27. What was `DOMRect` like in Firefox 27, did it not have any members?

Reply:
I've had a look in Firefox 27. `DOMRect` was exposed, and had members `left`, `top`, `right`, `bottom`, `width` and `height`. `document.body.getBoundingClientRect()` returned ` DOMRect`, and a `DOMRectList` was also exposed.

Reply:
I used http://software.hixie.ch/utilities/js/live-dom-viewer/ to check what happened before Firefox 27, and then `document.body.getBoundingClientRect()` would return a `ClientRect`. So 27 is when `ClientRect` was renamed to `DOMRect` and exposed, but still didn't have a constructor. So this change is correct.

Reply:
I've sent https://github.com/mdn/browser-compat-data/pull/8711 to add the `ClientRect` alt. name.